content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,98 @@ are designed for backwards compatibility rather than for best practice or maximu
 You may find that the default resources do not comply with the security policy on your Kubernetes cluster
 and in that case you can modify the installation configuration using Helm chart values to override the defaults.
 
+## Network Requirements and Network Policy
+
+The network requirements of each cert-manager Pod are summarized below.
+Some network requirements depend on specific Issuer / ClusterIssuer configurations
+and / or specific configuration options.
+
+When you have understood the network requirements of **your** cert-manager installation,
+you should consider implementing a "least privilege" network policy,
+using a [Kubernetes Network (CNI) Plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) such as Calico.
+
+The network policy should prevent untrusted clients from connecting to the cert-manager Pods
+and it should prevent cert-manager from connecting to untrusted servers,
+if there is a mis-configuration.
+
+An example of this recommendation is found in the Calico Documentation:
+> We recommend creating an implicit default deny policy for your Kubernetes pods, regardless if you use Calico or Kubernetes network policy. This ensures that unwanted traffic is denied by default.
+>
+> 📖 [Calico Best practice: implicit default deny policy](https://docs.tigera.io/calico/latest/network-policy/get-started/kubernetes-default-deny#best-practice-implicit-default-deny-policy).
+
+You can use the Kubernetes builtin `NetworkPolicy` resource,
+which is portable because it is recognized by any of the [Kubernetes Network (CNI) Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/).
+Or you may prefer to use the custom resources provided by your CNI software.
+
+> 📖 Learn about the [Kubernetes builtin NetworkPolicy API](https://kubernetes.io/docs/concepts/services-networking/network-policies/) and see some example policies.
```

Comment:
I'd want `see some example policies` to be in a link, either as part of the preceding link or as a link to another page.

Reply:
Done.